### DISCUSSION TITLE
Can't Find A Module

### DISCUSSION THREAD

**Discussion by Tansi-Jones** [mantinedev/mantine#2092] [Q&A] [answered]
Hello,
I tried importing the <CopyButton> component to my react app but it keeps saying "Module '"@mantine/core"' has no exported member 'CopyButton'.". But according to the documentation, it's to be imported from @Mantine/core, any help?

**Comment by rtivital** [comment, marked as answer]
Hi @Tansi-Jones, you need to update all `@mantine/` packages to the latest version. To update packages, run npx `npm-check-updates @mantine/* -u` then install dependencies with your package manager.

**Reply by Tansi-Jones** [reply]
Hello @rtivital I just updated my mantine packages but nothing happened

**Reply by rtivital** [reply]
What version do you have in your package.json? Have you installed dependencies after package.json was updated?

**Reply by Tansi-Jones** [reply]
@mantine/core: 4.2.12,
@mantine/hooks: "^4.2.12

**Reply by rtivital** [reply]
Latest version is 5.1.4, you need to update all `@mantine/` packages to it in order to use new features. You can find 4.x documentation here – http://v4.mantine.dev/

**Reply by Tansi-Jones** [reply]
Ohhh, when I ran "npx npm-check-updates @mantine/* -u" the first time it didn't update my packages.
Thank you @rtivital  Problem solved 🚀
